src/lib/Components/Inbox/Conversations/__tests__/ConversationSnippet-tests.tsx

```diff
@@ -40,6 +40,7 @@ const show = {
 }
 
 const artworkConversation = {
```

Comment:
Protip: If you annotate these variables as a specific type, VSCode will be able to give you even better guidance as to what props should be in here, rather than having to wait to see if the component accepts it.